modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -500,6 +500,7 @@
     <string name="episode_short_episode">E%d</string>
     <string name="episode_added_to_up_next">Added to up next</string>
     <string name="episode_removed_from_up_next">Removed from up next</string>
+    <string name="episode_status_up_next">Episode in up next queue</string>
```

Comment:
Do you think it might be worth dropping the word "episode" here so that it was just "In up next queue"? I was thinking that it might be obvious that this is an episode, so shortening the label could be a win. Just a thought, I don't feel strongly.

Reply:
Agreed, done.